src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --link-color: #00ACDC;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.nav {
+  padding: 0;
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  list-style: none;
   margin: 0;
 }
+
+
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 60px;
+  text-decoration: none;
+  margin-left: 20px;
+}
+
+.is-active {
```

Comment:
this helper selector should only have color and position

Reply:
not fixed 
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/ea59d23d-2659-4855-b801-54feaa0eedbb)